sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -34,6 +34,14 @@
 
 
 class FlinkRunner(portable_runner.PortableRunner):
+  def run_pipeline(self, pipeline, options):
+    portable_options = options.view_as(pipeline_options.PortableOptions)
+    if (options.view_as(FlinkRunnerOptions).flink_master in MAGIC_HOST_NAMES
+        and not portable_options.environment_type
+        and not portable_options.output_executable_path):
```

Comment:
This seems to be the only place where we use the parameter.

Reply:
Never mind. Found it.

Reply:
@robertwb What about the case where we have a pre-configured Flink job server with a custom master url? This will then force loopback execution although the user probably does not want that.